pkg/oauthserver/authenticator/password/keystonepassword/keystonepassword.go

```diff
@@ -6,8 +6,10 @@ import (
 	"runtime/debug"
 
 	"github.com/golang/glog"
+	"github.com/mitchellh/mapstructure"
```

Comment:
we have to vendor all dependencies. I would not add a dependency for this use.

Reply:
Isn't it already there? https://github.com/openshift/origin/tree/master/vendor/github.com/mitchellh/mapstructure

Reply:
ah, so it is. I still wonder if `result.ExtractUser()` does what you want already though